web/packages/teleport/src/Integrations/Enroll/MachineIDIntegrationSection.tsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright 2023 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  AnsibleIcon,
+  CircleCIIcon,
+  GitHubIcon,
+  GitLabIcon,
+  JenkinsIcon,
+  ServersIcon,
+} from 'design/SVGIcon';
+import { Box, Flex, Link as ExternalLink, Text } from 'design';
+import React from 'react';
+
+import {
+  IntegrationEnrollEvent,
+  IntegrationEnrollKind,
+  userEventService,
+} from 'teleport/services/userEvent';
+import { IntegrationTile } from 'teleport/Integrations/Enroll/common';
+
+interface Integration {
+  title: string;
+  link: string;
+  icon: JSX.Element;
+  kind: IntegrationEnrollKind;
+}
+
+const integrations: Integration[] = [
+  {
+    title: 'GitHub Actions',
+    link: 'https://goteleport.com/docs/machine-id/guides/github-actions/',
+    icon: <GitHubIcon size={80} />,
+    kind: IntegrationEnrollKind.MachineIDGitHubActions,
+  },
+  {
+    title: 'CircleCI',
+    link: 'https://goteleport.com/docs/machine-id/guides/circleci/',
+    icon: <CircleCIIcon size={80} />,
+    kind: IntegrationEnrollKind.MachineIDCircleCI,
+  },
+  {
+    title: 'GitLab CI/CD',
+    link: 'https://goteleport.com/docs/machine-id/guides/gitlab/',
+    icon: <GitLabIcon size={80} />,
+    kind: IntegrationEnrollKind.MachineIDGitLab,
+  },
+  {
+    title: 'Jenkins',
+    link: 'https://goteleport.com/docs/machine-id/guides/jenkins/',
+    icon: <JenkinsIcon size={80} />,
+    kind: IntegrationEnrollKind.MachineIDJenkins,
+  },
+  {
+    title: 'Ansible',
+    link: 'https://goteleport.com/docs/machine-id/guides/ansible/',
+    icon: <AnsibleIcon size={80} />,
+    kind: IntegrationEnrollKind.MachineIDAnsible,
+  },
+  {
+    title: 'Generic',
+    link: 'https://goteleport.com/docs/machine-id/getting-started/',
+    icon: <ServersIcon size={80} />,
+    kind: IntegrationEnrollKind.MachineID,
+  },
+];
+
+const MachineIDIntegrationSection = () => {
```

Comment:
Please use named export (if you want to know why named exports are better: https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad)

```suggestion
export const MachineIDIntegrationSection = () => {
```

Reply:
2ef44fb200